src/somesy/cli/sync.py

```diff
@@ -104,13 +104,13 @@ def sync(
     no_sync_julia: bool = typer.Option(
         None,
         "--no-sync-julia",
-        "-M",
-        help="Do not sync Project.toml(Julia) file",
+        "-K",
+        help="Do not sync Project.toml(Julia) file (default: False)",
     ),
     julia_file: Path = typer.Option(
         None,
         "--julia-file",
-        "-m",
+        "-k",
```

Comment:
Let's use `L`/`l` for Julia - `j` is already taken but the file extension for Julia is `.jl` so `l` would make sense... whereas there is no `k` in `Julia` ;)